packages/adp-tooling/templates/project/webapp/manifest.appdescr_variant

```diff
@@ -7,6 +7,9 @@
   "namespace": "apps/<%- app.reference %>/appVariants/<%- app.id %>/",
   "version": "0.1.0",
   "content": [
+<%if(app.content){-%><% for(let change of app.content){-%>
+    <%- JSON.stringify(change) %><%-","%>
```

Comment:
It might be nice to align the spacing of your JSON with the rest of the document e.g. JSON.stringify(change, undefined, 4)

Reply:
done